sim/warrior/arms/deadly_calm.go

```diff
@@ -0,0 +1,66 @@
+package arms
+
+import (
+	"time"
+
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/warrior"
+)
+
+func (war *ArmsWarrior) RegisterDeadlyCalm() {
+	if !war.Talents.DeadlyCalm {
+		return
+	}
+
+	dcActionID := core.ActionID{SpellID: 85730}
+
+	// Using an aura that modifies the PseudoStat instead of a SpellMod to avoid stomping
+	// over Battle Trance if it's procced. Having both of these edit spells could result in a sequence like
+	// 1: <Use Deadly Calm> (spell.CostMultiplier = 0)
+	// 2: <Proc Battle Trance> (spell.CostMultiplier = -1) (what does this even do? refund on every ability?)
```

Comment:
```Go
func (spell *Spell) ApplyCostModifiers(cost float64) float64 {
	cost -= spell.Unit.PseudoStats.CostReduction
	cost = max(0, cost*spell.Unit.PseudoStats.CostMultiplier)
	return max(0, cost*spell.CostMultiplier)
}
```
That's what it would do. So -1 wouldn't be that bad